v3.0-RC/gbfs.json

```
@@ -59,11 +60,10 @@
             },
             "required": ["name", "url"]
           }
```

What do you think about requiring at least 1 item in `feeds` array?

It would also improve consistency with the previous versions which require at least 1 property in `feeds`.

cc @josee-sabourin 
```suggestion
          },
          "minItems": 1
```

---

Why not :)